### PR TITLE
fix: do not initialize cubeAxesActor if not required

### PR DESCRIPTION
**Pull request by FezVrasta**
We don't need to initialize cubeAxesActor if the showCubeAxes prop is false

### CODE REVIEW

File: src/core/View.js [hunk 1]
```diff
@@ -467,15 +469,17 @@ export default class View extends Component {
...
     }
 
-    if (this.cubeAxes.setVisibility(showCubeAxes)) {
-      Array.from(this.cubeAxes.getActors()).forEach(({ setVisibility }) =>
-        setVisibility(showCubeAxes)
-      );
-      this.renderView();
-    }
+    if (this.props.showCubeAxes) {
+      if (this.cubeAxes.setVisibility(showCubeAxes)) {
```
Comment (floryst) [inline src/core/View.js:473]: Maybe do a deferred get call for the cube axes here, e.g. `if (this.getCubeAxes()...)`, where getCubeAxes will create the actor if `this.cubeAxes` is not set. That way we can support toggling showCubeAxes from a starting false state.

Comment (FezVrasta) [reply src/core/View.js:482]: Can you take a look at my latest changes?

File: src/core/View.js [hunk 2]
```diff
@@ -386,6 +392,11 @@ export default class View extends Component {
 
   componentDidUpdate(prevProps, prevState, snapshot) {
     this.update(this.props, prevProps);
+
+    // Cube Axes
+    if (this.props.showCubeAxes && this.cubeAxes == null) {
+      this.initCubeAxes();
+    }
```
Comment (floryst) [inline src/core/View.js:399]: This gets called after `this.update`, which means if `this.props.showCubeAxes` is set to true, [line 482](https://github.com/Kitware/react-vtk-js/pull/70/files#diff-b7267f7f1a27837a979426c002ada36d97925bce54b0388d65127314c21ff342R482) will try to deref `this.cubeAxes`, which will be null. This check should happen before the `if (this.props.showCubeAxes)` logic.

Comment (FezVrasta) [reply src/core/View.js:399]: Could you please update the PR yourself? I'm on PTO this week

**Pull request merged**